content/docs/command-reference/repro.md

```diff
@@ -175,9 +175,11 @@ up-to-date and only execute the final stage.
 
 ## Examples
 
-For simplicity, let's build a pipeline defined below. (If you want get your
-hands-on something more real, see this short
-[pipeline tutorial](/doc/start/data-pipelines)). It takes this `text.txt` file:
+To get hands-on experience with data science and machine learning pipelines, see
+[Get Started: Data Pipelines](/doc/start/data-pipelines).
```

Comment:
This whole change is out of scope but OK. Should we move this link up somewhere in the description, actually? Not in examples.

Reply:
p.s. It can stay here too if it makes less sense to have it in the desc.

Reply:
Yes, much better idea. Moved it up into the description and that should be better for SEO purposes too. PTAL

Reply:
Thanks. It feels to me like it's not in a natural place right now though. It interrupts the description flow. Maybe before **Parallel stage execution**? But again, it can stay in the top of **Examples** if there's no better place. It should also probably be a note (md block quote).

Reply:
> But again, it can stay in the top of Examples if there's no better place. It should also probably be a note (md block quote). 

OK @jorgeorpinel, submitted #1937 to fix this.